bin/export-db-to-sqlite.sh

```diff
@@ -63,6 +61,8 @@ tables_to_wipe_after_initial_migrate=(
 
 tables_to_wipe_after_import=(
     "auth_user"
+    "auth_user_groups"
+    "auth_user_user_permissions"
```

Comment:
This is where we need to reference those two tables because we do load in data for them as part of the import into sqlite, and we need to clear them out before uploading the DB to our bucket